### DISCUSSION TITLE
E se la fonte fosse un giornale?

### DISCUSSION THREAD

**Discussion by aborruso** [opendatasicilia/informa-covid19#55] [Generale]
Ciao,
ieri mi ha scritto Maurizio Foderà (membro storico di Open Data Sicilia) l'email di sotto.

Sembra che una testata giornalistica raccolga i dati di diversi comuni. Può - se usa il modello informacovid - diventare una delle fonti di progetto?

A mio avviso sì e partirei dal discutere su questo. 

Poi bisognerà sentirli, capire se sono disponibili in generale e se in particolare giornalmente vorranno usare le modalità informacovid.

E se sì, potremmo proporlo anche [a un giornale](https://www.facebook.com/groups/opendatasicilia/permalink/3860773150707978/) del comune di Messina.

Buona giornata

>Ciao Andrea
>Complimenti a te e a tutto il gruppo per il progetto Informacovid.
>
>L’ho seguito un po' in diretta e un po' in differita e comunque ho apprezzato le grandi potenzialità.
>
>Vado subito diretto alla domanda che mi è balenata in mente purtroppo in differita ed ecco il motivo di questa mail.
>
>Piuttosto che il singolo comune la dashboard potrebbe essere attivata da una testata giornalistica online che riceve il comunicato ufficiale da ASP ogni sera e che nel caso specifico è una testata non solo di Mazara ma che ha un network di comuni?
>
>Nello specifico:
>
>- Campobello
>- Castelvetrano
>- Marsala
>- Mazara
>- Partanna
>- Salemi
>- Trapani
> 
>
>La testata è https://www.primapaginamazara.it/ , https://www.primapaginacampobello.it/ , etc
>
>L’editore è un carissimo amico mio e penso potrebbe sposare il progetto.
> 
>
>Fammi sapere
>
>Grazie e a presto… è sempre un piacere ascoltarvi… 😉

**Comment by gpirrotta** [comment]
Ciao Andrea,
assolutamente sì.

InformaCOVID nasce per facilitare la vita ai comuni. Ciò non toglie che se un qualsiasi tipo di organizzazione raccoglie i dati COVID  per una città non possa creare un'istanza di InformaCOVID, ben vengano iniziative di questo tipo.

**Reply by aborruso** [reply]
Adesso dico a Maurizio di intermediare. Poi c'è da capire come gestire le info "tecniche".

**Reply by dennisangemi** [reply]
Buongiorno @aborruso , riapro questa discussione per capire se c'è qualche aggiornamento: Maurizio si è fatto sentire? Io in questo periodo ho molto tempo libero quindi potrei occuparmene (ho in mente un modo più rapido per aggiornare dashboard con più comuni, sacrifichiamo il form ma non credo sia molto efficace in casi come questi).

Gli step che mi piacerebbe seguire:

1. Mettermi in contatto con l'editore citato per verificare la disponibilità a sposare il progetto
2. Modificare il foglio di google sheet sulla base delle comunicazioni in excel che ricevono da ASP

**Reply by aborruso** [reply]
> Maurizio si è fatto sentire?

No. Trovi i suoi riferimenti credo cercando in mailing list ODS. Al limite scrivigli

**Reply by dennisangemi** [reply]
Fatto! Scriverò qui gli aggiornamenti. Grazie :)

**Comment by dennisangemi** [comment]
Che bello! Anche secondo me può diventare assolutamente una delle fonti di progetto. Forse dobbiamo pensare a un metodo più veloce per aiutare le testate a generare più sheets in un sol colpo?
